tckdb/backend/app/tests/endpoints/test_batch_upload.py

```diff
@@ -0,0 +1,312 @@
+import pytest
+
+from tckdb.backend.app.core.config import API_V1_STR
+from tckdb.backend.app.models.species import Species as SpeciesModel
+
+
+@pytest.mark.usefixtures("setup_database")
+class TestBatchEndpoint:
+    """
+    A class to test the batch upload endpoint
+    """
+
+    @pytest.fixture(scope="class", autouse=True)
+    def setup_payload(self, request, client):
+        """
+        A function to setup a payload
+        """
+        payload = {
+            # "authors": [
+            #     {
+            #     "connection_id": "temp_author_1",
+            #     "first_name": "Calvin",
+            #     "last_name": "Pieters",
+            #     "orcid": "0000-0001-6377-2161"},
+            #     {
+            #     "connection_id": "temp_author_2",
+            #     "first_name": "Florian",
+            #     "last_name": "Solbach",
+            #     "orcid": "0000-0003-1923-3747"}
+            # ],
+            "literature": [
+                {
+                    "connection_id": "temp_literature_1",
```

Comment:
also here for the hard coded data. if parts of the data should change according to specific test we can make this a util that return the data based on some args

Reply:
I am not so sure what you mean here.

Like make another script with just all the json code here as a function and then in this test, for example, import the function (which returns the json script)? Cause youre seeing this as this code will be used in other test cases?

Reply:
yes exactly, even it will be not used in another tests it will make our test a lot more readable and will help us know what we are actually checking